packages/swatch/swatch-group.md

```diff
@@ -217,6 +233,27 @@ When available, the value of `selected` will be constructed from the `value` att
 <div>Selected: [ "color-2", "color-1", "color-3" ]</div>
 ```
 
-## Swatch modifying attributes
+### Swatch modifying attributes
+
+An `<sp-swatch-group>` element can be modified by the following attributes/properties to customize its delivery as desired for your use case: `border`, `disabled`, `mixedValue` (accepted as the `mixed-value` attribute), `nothing`, `rounding`, `shape`, and `size`. Use these in concert with each other for a variety of final visual deliveries. Applying a value for one of these attributes/properties to an `<sp-swatch-group>` element will have it forward the value to all of the `<sp-swatch>` elements that are a direct child of the group, overriding any value that may be applied directly to those children. For example:
+
+```html
+<sp-swatch-group border="light" rounding="full" shape="rectangle" size="l">
+    <sp-swatch color="var(--spectrum-blue-500)"></sp-swatch>
+    <sp-swatch color="var(--spectrum-indigo-500)"></sp-swatch>
+    <sp-swatch color="var(--spectrum-purple-500)"></sp-swatch>
+</sp-swatch-group>
+```
+
+### Accessibility
+
+#### Best practices
+
+- Ensure swatches have sufficient color contrast for visibility.
+- Verify that swatches are appropriately labeled for screen readers.
```

Comment:
Can we expand on this any further? I'm assuming they would be labeled with `aria-label`?